readinglist/views/heartbeat.py

```diff
@@ -16,7 +16,6 @@ def get_heartbeat(request):
     status = dict(database=database)
     has_error = not all([v for v in status.values()])
     if has_error:
-        # XXX: use Cornice errors
         request.response.status = 503
```

Comment:
We don't want heartbeat endpoint to return a JSON error but just the heartbeat error format.